manifests/compose/validation/vm/kepler/etc/kepler/kepler.config/MODEL_SERVER_ENABLE

```diff
@@ -1 +1 @@
-true
+false
```

Comment:
why is this needed ?

Reply:
I figured that this is needed to enabled `kepler` -> `estimator` -> `model-server` scenario